docs/source-pytorch/common/trainer.rst

```diff
@@ -317,6 +317,10 @@ before any training.
 auto_select_gpus
 ^^^^^^^^^^^^^^^^
 
+.. warning:: ``auto_select_gpus`` has been deprecated in v1.9.0 and will be removed in v1.10.0.
```

Comment:
Normally we just remove mentions of deprecated code from the docs

Reply:
The entire section?

Reply:
Yes. All mentions of the deprecated functionality. This way new users don't even bother with deprecated code.